assets/section-footer.css

```diff
@@ -377,7 +377,7 @@ noscript .localization-selector.link {
 }
 
 @media screen and (min-width: 750px) {
-  .footer__copyright:not(.center) {
+  .footer__copyright {
```

Comment:
I think this needs to be `.footer__copyright:not(.footer__content-bottom-wrapper--center)` no ? 
Otherwise it always overwrite the text alignment on screens wider than 750px. 

There is a chance for this to happen: 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/12-06-4onzb-cvtoo.png)
</details>